dataproc/src/test/java/CreateClusterWithAutoscalingTest.java

```diff
@@ -0,0 +1,2 @@
+public class CreateClusterWithAutoscalingTest {
```

Comment:
Is this supposed to be empty? or missed in the commit?

Reply:
Oops, must've missed this while adding the correct version. I'll fix that...